cpp/ql/src/experimental/Security/CWE/CWE-1041/FindWrapperFunctions.qhelp

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>Finding for function calls for which wrapper functions exist.</p>
```

Comment:
I think this section should say _why_ you want to find function calls for which wrappers exist. What implication does this have for security?

Reply:
The basic idea behind security is that if a developer creates a wrapper function with additional checks, he believes that this is necessary to improve security. And that's a problem if the code contains a non-wrapped call, both from the support side and from a security point of view.
Unfortunately, I have not yet formulated the description, it sounds like this.

`The presence of a shell function with additional check indicates the possible risks of the call. Use this check everywhere.`